modules/creating-image-expiration-notification.adoc

```diff
@@ -0,0 +1,82 @@
+// module included in the following assemblies:
+
+// * use_quay/master.adoc
+// * quay_io/master.adoc
+
+:_content-type: CONCEPT
+[id="creating-image-expiration-notification"]
+= Creating an image expiration notification
+
+Image expiration event triggers can be configured to notify users through email, Slack webhooks, and so on, and can be configured at the repository level. Triggers can be set for images expiring in any amount of days, and can work in conjunction with the auto-pruning feature.
+
+Image expiration notifications can be set by using the {productname} v2 UI or by using the `createRepoNotification` API endpoint.
+
+.Prerequisites
+
+* You have set `FEATURE_GARBAGE_COLLECTION: true` in your `config.yaml` file.
```

Comment:
optional. as by default it is true https://github.com/quay/quay/blob/a06b6e929c7a38b78ba885fa2dfc425f6b995dc1/config.py#L729
@Sunandadadi wdyt?

Reply:
instead,  `FEATURE_AUTO_PRUNE: true` looks a must

Reply:
Added/slightly reworded.

Reply:
@SeanZhao-redhat 
`FEATURE_GARBAGE_COLLECTION` is true by default but it is a must for this feature. 
`FEATURE_AUTO_PRUNE` is not a must.